ui/components/app/permissions-connect-permission-list/index.scss

```diff
@@ -1,8 +1,9 @@
 .permissions-connect-permission-list {
+  width: 100%;
```

Comment:
Unsure why this was needed :thinking:

Reply:
@hmalik88

Reply:
<img width="300" alt="Screen Shot 2022-01-26 at 2 23 20 PM" src="https://user-images.githubusercontent.com/41640681/151232145-79b928c4-4936-4cb7-b58d-cad93dae6ade.png">

Reply:
Hmm reverted the change and everything was fine....unless I'm missing something @ritave?